pkg/odo/cli/cli.go

```diff
@@ -272,6 +272,5 @@ func ShowHelp(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
-	_ = cmd.Help()
```

Comment:
I think we can have a integration test where we check for only one occurence of `Invalid command - see available commands/subcommands above`

Reply:
Yes, may be it will be helpful to avoid future regression.

Reply:
added the test - I couldn't use `Invalid command - see available commands/subcommands above` as that shows up twice already and hence went for a unique string